### PR TITLE
CB-9811: Configure Ranger with custom cloud storage path for HBase da…

### PR DESCRIPTION
**Pull request by LinaAtAustin**
…ta at DataLake

### CODE REVIEW

File: template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/ranger/RangerCloudStorageServiceConfigProvider.java [hunk 1]
```diff
@@ -58,6 +65,12 @@
                     .orElseGet(() -> cloudPaths.add(RANGER_AUDIT.name() + "=" + getDefaultRangerAuditUrl(templateProcessor, templatePreparationObject)));
 
 
+            if (datalakeCluster) {
```
Comment (lajosrodek) [inline template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/ranger/RangerCloudStorageServiceConfigProvider.java:68]: I agree making this check is a good idea in general, but this config provider is explicitly for Ranger Admin that is supposed to exist in one place only, namely in the DL (or multiple DLs if we ever happen to support that in any environment). See `isConfigurationNeeded()` below, it explicitly checks for the presence of Ranger Admin in the cluster template, so the rest of this config provider (including the encompassing `getServiceConfigs()` method) will not be executed for DH clusters.

In short: I suggest removing the additional check for the cluster kind and let it just assume DL.

File: template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/ranger/RangerCloudStorageServiceConfigProviderTest.java [hunk 2]
```diff
@@ -164,16 +167,22 @@ public void defaultRangerHdfsAuditUrlWithSingleNamenode() {
...
+            locations.add(new StorageLocationView(buildStorageLocation("hbase.rootdir",
+                    "abfs://data@your-san.dfs.core.windows.net/hbase")));
         }
         AdlsGen2FileSystemConfigurationsView fileSystemConfigurationsView =
                 new AdlsGen2FileSystemConfigurationsView(new AdlsGen2FileSystem(), locations, false);
 
         GeneralClusterConfigs generalClusterConfigs =  new GeneralClusterConfigs();
         generalClusterConfigs.setPrimaryGatewayInstanceDiscoveryFQDN(Optional.of("fqdn"));
 
+        SharedServiceConfigsView sharedServicesConfigsView = new SharedServiceConfigsView();
```
Comment (lajosrodek) [inline template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/ranger/RangerCloudStorageServiceConfigProviderTest.java:179]: This thing might not be needed as per my earlier comment.

File: template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/ranger/RangerCloudStorageServiceConfigProviderTest.java [hunk 3]
```diff
@@ -201,9 +212,13 @@ public void defaultRangerHdfsAuditUrlWithSingleNamenode() {
...
```
Comment (lajosrodek) [inline template-manager-cmtemplate/src/test/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/ranger/RangerCloudStorageServiceConfigProviderTest.java:215]: Not needed this one either.

**Review by lajosrodek — APPROVED** [review]
LGTM, only some minor comments.

**Comment by LinaAtAustin** [comment]
@lajosrodek @keyki I have made update based on feedback. All tests past. Can you merge my change? Thanks!

**Review by kmanamcheri — COMMENTED** [review]
Please update the commit message to state what testing you have done.

**Comment by lajosrodek** [comment]
Thanks, looks good.

**Review by kmanamcheri — APPROVED** [review]
